### DISCUSSION TITLE
Bulk relations connect

### DISCUSSION THREAD

**Discussion by LuizTokuhara** [prisma/prisma#8173] [Q&A] [answered]
Is it possible to connect to multiple relations using "connect" ?

Something like:

```ts
const updateAuthor = await prisma.user.update({
  where: {
    id: 20,
  },
  data: {
    posts: {
      connect: [ {id: 1}, {id: 2}, {id: 3} ]
    },
  },
})
```

**Comment by ryands17** [comment, marked as answer]
Yes it is.

**Reply by LuizTokuhara** [reply]
Thanks for reply

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
